library/components/TrendArrow.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="trendArrow">
+        <arrow-up-icon v-if="difference > 0" :size="iconSize" />
+        <minus-icon v-else-if="difference === 0" :size="iconSize" />
+        <arrow-down-icon v-else :size="iconSize" />
+    </div>
+</template>
+
+<script>
+export default {
+  name: "TrendArrow",
```

Comment:
Are we appending `T` to all topcoat-public components?

Reply:
...yes, yes we are. :joy: Will fix!